eq-publisher/src/eq_schema/Question.test.js

```diff
@@ -110,6 +110,59 @@ describe("Question", () => {
     });
   });
 
+  describe("additional information", () => {
+    describe("when there is content", () => {
+      it("should be populated when both label and content", () => {
+        const question = new Question(
+          createQuestionJSON({
+            additionalInfoLabel: "additionalInfo label",
+            additionalInfoContent: "<p>additionalInfo content</p>",
+          })
+        );
+        expect(last(question.answers).guidance).toBeDefined();
+        expect(last(question.answers).guidance.show_guidance).toBeDefined();
+        expect(last(question.answers).guidance.hide_guidance).toBeDefined();
+        expect(last(question.answers).guidance.content).toBeDefined();
+      });
+      it("should be populated when label and no content", () => {
```

Comment:
Do we ever want answer guidance with no content?

Reply:
IMO, no it doesn't make sense and this came up during the development of question definitions as well.  It's valid schema so we decided not to introduce any ad-hoc validation.

Reply:
I checked in runner and if there is no label it doesn't show no matter if there is content.

Reply:
Ok.  Would need input from @hamishtaplin regarding this as it changes AC.  We should apply same concepts to definitions as well.  It would make sense to change the preview to match what runner does.